extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -368,12 +368,16 @@
 		_t.image = parentNode.querySelector("img");
 		_t.updateInterval = parseInt(parentNode.getAttribute("data-update-interval"), 10);
 
-		if (_t.updateInterval === 0) {
-			return;
-		}
-
 		_t.url = _t.image.getAttribute("src").replace(/\d+$/, "");
 
+        _t.setValuePrivate = function() {
```

Comment:
I probably should have added this as eslint warning, but in the rest of the file tabs are used for indentation.